web/src/app/util/Chips.tsx

```diff
@@ -27,18 +27,18 @@ export function ServiceChip(props: WithID<ChipProps>): JSX.Element {
   const { id, label, ...rest } = props
   const navigate = useNavigate()
 
-  const { data, loading, error } = useQuery(serviceQuery, {
+  const [{ data, fetching, error }] = useQuery({
+    query: serviceQuery,
     variables: {
       id,
     },
-    skip: Boolean(label),
-    fetchPolicy: 'cache-first',
-    pollInterval: 0,
```

Comment:
is this the default poll interval now unless set? will the config overwrite that if so?

Reply:
`pollInterval: 0` was our way of disabling polling before. With `urql` if we set `cache-first` it will only ever re-fetch on window refocus, or cache invalidation, but not on time-updates.

Put another way, instead of `cache-first` and `pollInterval: 0` we can just set `cache-first` with the new stuff

Reply:
Sounds good, thanks!